packages/integrations/node/src/serve-app.ts

```diff
@@ -0,0 +1,25 @@
+import { NodeApp } from "astro/app/node"
+import type { RequestHandler } from "./types.js";
+
+/**
+ * Intended to be used in both standalone and middleware mode.
```

Comment:
We could document what it does, other than where :)

Reply:
Added :)

https://github.com/withastro/astro/pull/9582/files#diff-e93f38dcd3505fbeb1c5979bca34bc90e48b1fe9b1f3ead1102c31993e5ef286R5-R6